pkg/client/results/testdata/mockResults/plugins/ds-junit-01/results/global/output.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+  <testsuite tests="204" failures="0" time="0.0574002">
```

Comment:
No, I could actually remove the files that dont matter (actual logs/results). The only thing that matters is the summary.

Reply:
Sorry, I misunderstood the question.

No it doesn't matter but one day it actually might. For now I'll leave it as is so that it is even discernable where the data comes from.

I don't use those values (I manually iterate over the data) so that I can get passed/failed/skipped whereas this only shows tests (total) and failed.